jobs/test_e2e_errand/templates/bin/run.erb

```diff
@@ -15,40 +16,63 @@ export PATH="/var/vcap/packages/java8/bin:$PATH"
 
 [ ! -e /home/vcap/.sincedb ] || rm /home/vcap/.sincedb
 
-
-echo '' ; echo '==> Purging existing test data...'
-
-curl -s -X DELETE '<%= p('test_e2e_errand.elasticsearch_hostport') %>/_all/sample-logs--repo-commits' ; echo ''
```

Comment:
Aha - from your comment below I see you are expecting https://github.com/logsearch/logsearch-boshrelease/blob/develop/src/sample-logs/repo-commits/logstash-filters.conf to be installed - which I don't currently have